SQF/dayz_code/stringtable.xml

```diff
@@ -7475,7 +7475,7 @@
 		</Key>
 		<Key ID="str_death_shothead">
 			<English>a gunshot to the head.</English>
-			<German>durch einen Schuss in den Kopf.</German>
+			<German>durch einen Kopfschuss.</German>
```

Comment:
Okay, I'll change that back in the next commit. Actually I can't continue because of some exams that I currently have. I'll continue asap when I have more time.